testsuite/python/lb_vtk.py

```diff
@@ -0,0 +1,225 @@
+#
+# Copyright (C) 2010-2020 The ESPResSo project
+#
+# This file is part of ESPResSo.
+#
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+import unittest as ut
+import unittest_decorators as utx
+
+import os
+import numpy as np
+
+try:
+    import vtk
+    from vtk.util import numpy_support as VN
+    skipIfMissingPythonPackage = ut.case._id
```

Comment:
what's that?

Reply:
`vtk` is not a hard dependency

Reply:
but what is ```_id```

Reply:
not sure if it's a good idea to use a private variable

Reply:
that's just a lambda that returns the input function, it's called `_id` because `id` is a python built-in function

Reply:
yes but if it was part of the official interface they'd looked for a proper name, right?

Reply:
we now use our own `_id` lambda in 897c5bb118